pkg/apis/v1beta1/labels.go

```diff
@@ -46,7 +46,6 @@ const (
 const (
 	DoNotDisruptAnnotationKey             = apis.Group + "/do-not-disrupt"
 	ProviderCompatibilityAnnotationKey    = apis.CompatibilityGroup + "/provider"
-	ManagedByAnnotationKey                = apis.Group + "/managed-by"
```

Comment:
nit. 

Do we need to drop it from the v1beta1? I think it makes sense to leave it there, since that will be a breaking change

Reply:
What part does this break? The garbage collection is now being done via `NodePoolLabelKey` which i present on both v1beta1 and v1.